docker-compose.yaml

```diff
@@ -1,32 +1,32 @@
-version: '2'
+version: '3.2'
 
 services:
-  service_php:
+  php:
     build: docker/php
-    tty: true
-    labels:
-      kompose.service.type: nodeport
     ports:
       - 9000:9000
     volumes:
       - ./quarto:/quarto
-
-  service_nginx:
+    depends_on:
+      - postgres
+  nginx:
     image: nginx:alpine
     ports:
       - "8081:8081"
     volumes:
       - ./quarto:/quarto
       - ./docker/nginx/site.conf:/etc/nginx/conf.d/default.conf
-
-  service_postgres:
+    depends_on:
+      - php
+  postgres:
     image: postgres:9.6
     environment:
-      - POSTGRES_USER=docker
-      - POSTGRES_PASSWORD=docker
+      - POSTGRES_USER=quatro
+      - POSTGRES_PASSWORD=quatro
+      - POSTGRES_DB=quatro
```

Comment:
`POSTGRES_DB` was missing.
Later, it might be a good idea to use environment variables (with [direnv](https://direnv.net/) for example), or then with a configuration file : 
```
env_file:
            - ./config/${NODE_ENV}.env
```